lms/djangoapps/grades/tasks.py

```diff
@@ -33,6 +33,7 @@
 log = getLogger(__name__)
 
 KNOWN_RETRY_ERRORS = (DatabaseError, ValidationError)  # Errors we expect occasionally, should be resolved on retry
+RECALCULATE_GRADE_DELAY = 2  # in seconds, to prevent excessive _has_db_updated failures. See TNL-6424.
```

Comment:
It seems to me that since this delay should only be applied to the task contained in this file, this is where it should be defined. Anyone else have a better suggestion for location, or prefer a different name?